packages/aws-cdk/test/test.init.ts

```diff
@@ -45,6 +45,18 @@ export = {
     test.equal(true, await fs.pathExists('package.json'));
     test.equal(true, await fs.pathExists('bin'));
 
+    test.done();
+  },
+
+  async 'git directory does not throw off the initer!'(test: Test) {
+    fs.mkdirSync('.git');
+
+    await cliInit('app', 'typescript', false);
```

Comment:
Will this run npm install during the test?

Reply:
Apparently it does not.

Reply:
How come?